x-pack/plugins/cloud/kibana.json

```diff
@@ -7,7 +7,7 @@
   "version": "8.0.0",
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "cloud"],
-  "optionalPlugins": ["cloudExperiments", "usageCollection", "home", "security"],
+  "optionalPlugins": ["usageCollection"],
```

Comment:
FWIW, I was really tempted to move the usage collector outside (I was thinking `kibana_usage_collection`, but `cloud_usage_collection` could do as well). However, I didn't see a reason for `usageCollection` to ever depend on on the `cloud`, so I thought it could live there for now.